securedrop_client/logic.py

```diff
@@ -1066,13 +1066,19 @@ def delete_conversation(self, source: db.Source) -> None:
     @login_required
     def download_conversation(self, id: state.ConversationId) -> None:
         files = self._state.conversation_files(id)
+        file_count = len(files)
+        download_count = 0
         for file in files:
             if not file.is_downloaded:
+                download_count += 1
                 job = FileDownloadJob(str(file.id), self.data_dir, self.gpg)
                 job.success_signal.connect(self.on_file_download_success)
                 job.failure_signal.connect(self.on_file_download_failure)
                 self.add_job.emit(job)
                 self.file_download_started.emit(file.id)
+        logger.debug(
+            f"Downloaded {download_count} files, {file_count - download_count} were already downloaded (total: {file_count} files)"  # noqa: E501
+        )
```

Comment:
(I have no big issue with the `file_count` and `download_count` stuff here, but just want to make sure you intended for it to be in prod and it's not just being used for debug purposes)

Reply:
Being a `logger.debug` log, I it will only be emitted when the debug log level is active. With that context, yes, it was the intention keeping it longer-term. (Was that not your intention? We can remove it as well, I don't foresee a strong _need_ for it either.)